src/linkedList.js

```diff
@@ -0,0 +1,131 @@
+"use strict"
+import Node from '../src/node'
+export default class LinkedList {
+  constructor(data){
+    this.head = null
+    this.tail = null
+    this.size = 0
+
+  }
+
+  getHeadNode(){
+    if (!this) {
+      return null
+    }
+    return this.head
+  }
+
+  getTailNode(){
+    if (!this) {
+      return null
+    }
+    return this.tail
+  }
+
+  contains(string){
+    let current = this.head
+    if (this.head.data == string) {
```

Comment:
If I have 10 things in my list will this still be right? seems like it's checking the first two levels....